setup.py

```diff
@@ -43,8 +43,8 @@ def run_tests(self):
         'flake8-import-order>=0.18',
         'flake8-quotes',
         'pep8-naming>=0.7.0',
-        'pycodestyle>=2.3.1',
-        'pytest>=3.6.4',
+        'pycodestyle==2.3.1',
+        'pytest==3.6.4',
```

Comment:
Yes both pinned versions were for a reason:
pycodestyle >= 2.4.0 is incompatible with flake8 3.5.0
pytest >= 3.6.4 is incompatible with pytest-runner and/or flake8 I believe